build/WinObjC.Frameworks.Dependencies/Readme.txt

```diff
@@ -0,0 +1,9 @@
+﻿# WinObjC.Frameworks.UWP Readme
+
+Building this project will create a .nupkg containing just this Readme file. 
+
+It's always a good idea to provide a more useful package to your users, 
+alongside a nice descriptive Readme like this one that will open automatically
+when the add your package to a project.
+
+Happy nugetizing!
```

Comment:
yeah.

---
In reply to: [102594711](https://github.com/Microsoft/WinObjC/pull/2034#discussion_r102594711) [](ancestors = 102594711)